ivy/functional/ivy/experimental/linear_algebra.py

```diff
@@ -172,7 +170,6 @@ def eigh_tridiagonal(
 def diagflat(
     x: Union[ivy.Array, ivy.NativeArray],
     /,
-    *,
```

Comment:
why remove the asterisk (*)? Arent the remaining parameters keyword-only? The docstring examples are passing the `offset` argument as a kwarg